vuu-ui/showcase/src/examples/Apps/NewTheme.examples.tsx

```diff
@@ -93,75 +93,17 @@ const ShellWithNewTheme = () => {
     ];
   }, [handleCloseDialog, handleSave]);
 
-  //TODO what the App actually receives is an array of layouts
-  const layout = useMemo(() => {
-    return {
-      type: "Stack",
-      props: {
-        className: "vuuShell-mainTabs",
-        TabstripProps: {
-          allowAddTab: true,
-          allowRenameTab: true,
-          animateSelectionThumb: false,
-          location: "main-tab",
-        },
-        preserve: true,
-        active: 0,
-      },
-      children: [
-        {
-          type: "Stack",
-          props: {
-            active: 0,
-            title: "My Instruments",
-            TabstripProps: {
-              allowRenameTab: true,
-              allowCloseTab: true,
-            },
-          },
-          children: [
-            {
-              type: "View",
-              props: {
-                title: "European Stock",
-              },
-              style: { height: "calc(100% - 6px)" },
-              children: [
-                {
-                  type: "AutoTableNext",
-                },
-              ],
-            },
-            {
-              type: "View",
-              props: {
-                title: "Other Stock",
-              },
-              style: { height: "calc(100% - 6px)" },
-              children: [
-                {
-                  type: "AutoTableNext",
-                },
-              ],
-            },
-          ],
-        },
-      ],
-    };
-  }, []);
-
   return (
     <ContextMenuProvider
       menuActionHandler={handleMenuAction}
       menuBuilder={buildMenuOptions}
     >
       <Shell
-        defaultLayout={layout}
         leftSidePanelLayout="full-height"
         leftSidePanel={<LeftNav style={{ width: 240 }} />}
         loginUrl={window.location.toString()}
         user={user}
-        saveLocation="local"
+        saveUrl="http://127.0.0.1:3000/api/vui/layouts"
```

Comment:
need to be a little careful with the ShowCase examples. Bear in mind that these are published to CloudFlare pages, where we won't have any server running. It's fine to have stories that require a server - we have plenty that require Vuu. BUT we must handle the scenario where the server isn't available and show an error message.
Also , keep one version of the NewTheme app as a purely standalone example. Ideally, we can have a single example, which can be extended with a second example which provides a saveUrl, so that example would work with the remote REST service

Reply:
Actually, now I'm looking at Peter's PR (#40) and thinking about the problem again, I imagine an implementation which covers these comments will be handled by the interface?

Reply:
I think you are right Cara: the saveLocation should be with the LayoutPersistenceManager